quarkchain/cluster/master.py

```diff
@@ -891,6 +893,10 @@ def __has_all_shards(self):
                 if slave.has_shard(full_shard_id):
                     self.branch_to_slaves.setdefault(full_shard_id, []).append(slave)
 
+        for grpc_slave in self.cluster_config.GRPC_SLAVE_LIST:
```

Comment:
this may be moved to constructor since it doesn't depend on normal slaves.

also, one-liner:

```python
self.grpc_slave_pool = [GrpcClient(s.HOST, s.PORT) for s in self.cluster_config.GRPC_SLAVE_LIST]
```